File: java-demo-programs/MultiplicationTableDemo.java
public class MultiplicationTableDemo {
    public static void main(String[] args) {
        int tableNumber = 7;

        System.out.println("Multiplication table for " + tableNumber + ":");
        for (int multiplier = 1; multiplier <= 10; multiplier++) {
            int product = tableNumber * multiplier;
            System.out.println(tableNumber + " x " + multiplier + " = " + product);
        }
    }
}

File: python-demo-programs/multiplication_table_demo.py
"""Display a multiplication table for an example number."""


def print_multiplication_table(number: int, limit: int = 10) -> None:
    """Print a multiplication table from 1 through the supplied limit."""
    print(f"Multiplication table for {number}")

    for multiplier in range(1, limit + 1):
        product = number * multiplier
        print(f"{number} x {multiplier} = {product}")


def main() -> None:
    """Print the first ten multiples of seven."""
    print_multiplication_table(7)


if __name__ == "__main__":
    main()